exist-core/src/main/java/org/exist/jetty/JettyStart.java

```diff
@@ -298,25 +286,24 @@ public synchronized void run(final String[] args, final Observer observer) {
                 
                 if (handler instanceof ContextHandler) {
                     final ContextHandler contextHandler = (ContextHandler) handler;
-                    logger.info("\t{}", contextHandler.getContextPath());
+                    logger.info("{}", contextHandler.getContextPath());
                 }
-                
-                if (iprange != null) {
-                    if (handler instanceof ServletContextHandler) {
-                        final ServletContextHandler contextHandler = (ServletContextHandler) handler;
-                        contextHandler.addServlet(new ServletHolder((Class<? extends Servlet>) iprange), "/iprange");
 
-                        String suffix;
-                        if (contextHandler.getContextPath().endsWith("/")) {
-                            suffix = "iprange";
-                        } else {
-                            suffix = "/iprange";
-                        }
+                if (handler instanceof ServletContextHandler) {
 
-                        logger.info("'" + contextHandler.getContextPath() + suffix + "'");
-                    }
+                    ServiceLoader<ExistExtensionServlet> load = ServiceLoader.load(ExistExtensionServlet.class);
+                    load.forEach(existExtensionServlet ->
+                    {
+                        final ServletContextHandler contextHandler = (ServletContextHandler) handler;
+                        final String pathSpec = existExtensionServlet.getPathSpec();
+                        final Servlet servlet = (Servlet) existExtensionServlet;
```

Comment:
I think `ExistExtensionServlet` should like extend `Servlet` so that we can avoid this cast which could fail otherwise.

Reply:
lemme try again